android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -358,7 +364,7 @@ private void syncComplete() {
         OpenEventApp.postEventOnUIThread(new EventLoadedEvent(event));
         saveEventDates(event);
 
-        downloadPageId();
+        storeFeedDetails();
```

Comment:
This file should only feature changes here. The reason for this is also very future specific. If someone from future wants to see who changed these lines using git blame, they'll see your name all over the place where you didn't even write the code. Now you wouldn't want blame on things you did not do, would you? :wink: